.github/ISSUE_TEMPLATE/onboarding_to_cluster.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Onboard to a cluster
+about: Requesting namespaces and/or user onboarding to a managed OpenShift cluster
+title: ""
+labels: onboarding
+assignees: tumido, HumairAK
```

Comment:
Shouldn't assignees actually be empty? This way any two OWNERS of the repo can onboard users instead of being blocked on just the two of you.

Reply:
Will change that! :+1: Good point